spec/stop_visits.schema.json

```diff
@@ -133,7 +133,7 @@
       }
     },
     {
-      "name": "load",
+      "name": "vehicle_load",
```

Comment:
I'm ok with vehicle load, but wonder if "passenger_load" might be more appropriate

Reply:
I would be OK with both.  `vehicle_load` is useful to distinguish from platform or escalator loads, all of which are passenger loads, but in the context of the stop_visits table it's clear that passenger_load would refer to the load in the vehicle.  What isn't clear is if the load is measured upon arriving at the stop or after departing it.  If we want the column name to be self-documented we could consider `departure_load` or `arrival_load`.

Reply:
+1 for `departure_load`

Reply:
I prefer `departure_load` too, or if we want shorter, easier to type names, `load_out`, which I admit is less clear.

Reply:
Hi all, I pulled `vehicle_load` from the field description, but definitely support other options. For `departure_load`, is it potentially confusing given that some systems will transmit an arrival load at the terminal stop? Happy if we define it specifically as departure load too (so you know arrival load at the terminal would be departure load at the second to last stop).

Reply:
Good point. The description should probably be updated to define this as the load at arrival for the final stop of a trip and departure for all others. Are there systems that provide only arrival loads? Do we need `arrival_load` and `departure_load` fields?

Reply:
Sounds like the simplest answer is to use `departure_load` and document that it means the load at departure from the stop in the description. I agree that converting between arrival and departure load is pretty trivial, so lets just leave that out, since I don't think the description is a good place for it.